semcore/breadcrumbs/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [5.29.1] - 2024-04-30
+
+### Fixed
+
+- Some a11y improvements.
```

Comment:
We should provide more detailed changelog

Reply:
changed